src/web/lib/ArticleRenderer.tsx

```diff
@@ -46,6 +49,20 @@ const commercialPosition = css`
     position: relative;
 `;
 
+const unsafeEmbedWrapperStyles = css`
```

Comment:
Is this an odd place for these styles? Seems very high in the render, can the figure not be inside the `UnsafeEmbeBlockComponent`?